src/workerd/io/worker.c++

```diff
@@ -1018,6 +1018,9 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
 
     lock->setCaptureThrowsAsRejections(features.getCaptureThrowsAsRejections());
     lock->setCommonJsExportDefault(features.getExportCommonJsDefaultNamespace());
+    if (features.getNodeJsCompatV2()) {
+      lock->setNodeJsCompatEnabled();
```

Comment:
Should `setNodeJsCompatEnabled` mention the "v2"?

Reply:
No, eventually the goal is to combine the two compat flags if we can. In either case, there's no reason to version this particular API.